dnsapi/dns_linode_v4.sh

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
+
+#Original Author: Philipp Grosswiler <philipp.grosswiler@swiss-design.net>
+#v4 Update Author: Aaron W. Swenson <aaron@grandmasfridge.org>
+
+LINODE_V4_API_URL="https://api.linode.com/v4/domains"
+
+########  Public functions #####################
+
+#Usage: dns_linode_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_linode_add() {
```

Comment:
Do the DNS functions need to have the `_v4` suffix added, now that the filename is changed from `dns_linode.sh` → `dns_linode_v4.sh`..?

One person had troubles with this:

````
[date] It seems that your api file is not correct, it must have a function named: dns_linode_v4_add
[date] Please check log file for more details: /tmp/acme/[domain name]/acme_issuecert.log
[date] It seems that your api file doesn't define dns_linode_v4_rm
[…]
```

Reply:
sorry, fixed, please try again.